plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/DummyMockitoTest.java

```diff
@@ -0,0 +1,23 @@
+package com.google.cloud.tools.eclipse.appengine.localserver;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.notNullValue;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
There was a few classloader issue, and the result was that probably the @Test annotation was pulled in from a different bundle than the annotation finder code, and hence the annotation was not recognized.